### ISSUE TITLE
Server stats not displaying on main page under admin account

### ISSUE THREAD

**Issue by tarkh** [mongo-express/mongo-express#743]
MongoDB 5.0.1. For some reason I'm getting `Turn on admin in config.js to view server stats!` message on main page, while `ME_CONFIG_MONGODB_ENABLE_ADMIN="true"`. After some digging in to the code, I've fix this by patch below. Is there something wrong with my setup?

```diff
diff --git a/lib/router.js b/lib/router.js
index e8061db..97dc577 100644
--- a/lib/router.js
+++ b/lib/router.js
@@ -249,7 +249,7 @@ const router = async function (config) {
   // mongodb mongoMiddleware
   const mongoMiddleware = function (req, res, next) {
     req.mainClient = mongo.mainClient;
-    req.adminDb = mongo.adminDb;
+    req.adminDb = mongo.adminDb || mongo.clients[0].adminDb;
     req.databases = mongo.getDatabases(); // List of database names
     req.collections = mongo.collections; // List of collection names in all databases
     req.gridFSBuckets = utils.colsToGrid(mongo.collections);
```

**Comment by SGA-max-faxalv** [comment]
As #792 is now merged, this issue should be possible to close?

**Comment by Foorack** [comment]
As #792 is now merged, this issue should be possible to close?

**Comment by Foorack** [comment]
@mzch Also this one please. This is a duplicate of #647, which now has been solved by #792. Thank you.

**Comment by shakaran** [comment]
Closing this as fixed, please reopen if still present
